### DISCUSSION TITLE
Novo no vuejs

### DISCUSSION THREAD

**Discussion by marcosmarchette** [frontendbr/forum#2091] [Perguntas e Respostas]
Pessoal, boa tarde
Sou novo no vuejs e estou com uma duvida
Tenho um array  dentro de outro array no vuejs
Ate ai tudo bem
Gostaria de renderizar as  imagens que estão incluídas nesse array na tag <img>
Esse array tem um cabeçalho e detalhes...As imagens estão no detalhes
Estou enviando o arquivo vue para referencia

Se alguém puder me dar uma ajuda eu agradeço

OBS: As imagens ate aparece porem ela duplica

Abraços a todos

[principal.zip](https://github.com/frontendbr/forum/files/6548230/principal.zip)

**Comment by igorlvicente** [comment]
Se você colocar o arquivo em um gist, acho que vai ser mais fácil te ajudar.

**Comment by wilsonneto-swe** [comment]
Manda a parte do código ai, por favor, nos ajude a te ajudar...

**Comment by marcosmarchette** [comment]
Segue conforme o @igorlvicente  solicitou
<script src="https://gist.github.com/marcosmarchette/8e0822680a899d67f4460807a3264e34.js"></script>

**Comment by wmartins** [comment]
@marcosmarchette, baixei o zip que tu enviou aqui para tentar entender e te ajudar, o ideal seria ter uma forma de vermos o problema em si, talvez até ter um jeito de executar nas nossas próprias máquinas, mas entendo que pode ser difícil de fazer essa configuração, então vou tentar ajudar com base nisso aqui:

> OBS: As imagens ate aparece porem ela duplica

## Possibilidade 1

Me parece que nesse trecho aqui tu tá sempre adicionando a mesma referência para o campo imagens:

```js
for (let i = 0; i < retorno.length; i++) {
  this.rifas.push({
    descricao: retorno[i].descricao,
    dataCriacao: moment(retorno[i].dataCriacao).format("DD/MM/YYYY"),
    dataSorteio: retorno[i].dataSorteio,
    valorRifa: retorno[i].valorRifa,
    numeros: retorno[i].numeros,
    imagens: this.cotas,
  });
}
```

No caso, acho que tu precisaria só ajustar para pegar a imagem correta:

```js
{
  // ...
  imagens: this.cotas[i]
  // ...
}
```

E também alterar o template para usar a imagem correta:

```vue
<img
  v-bind:src="rifa.imagens.url"
  :alt="rifa.imagens.alt"
  :key="rifa.imagens.id"
  width="200"
/>
```

Não sei se isso ajuda, e caso não seja, manda mais informações (se conseguir gravar um gif/vídeo do problema ocorrendo e o que tu esperava que acontecesse) para podermos ajudar.
